src/routes/tooltips/+page.svelte

```diff
@@ -40,11 +47,14 @@
 	</section>
 	<section class="space-y-8">
 		<h2 class="h2">Open state</h2>
```

Comment:
Code Block components are not adhering to `space-y-8` in the preview:

![image](https://github.com/skeletonlabs/floating-ui-svelte/assets/1509726/558dd373-dfb5-439f-abdd-75cbc3268bdc)

It looks like this is because the surrounding Code Block template parent element (the div) uses `display: contents`:
https://tailwindcss.com/docs/display

I'd recommend against this if possible, or moving it into a child wrapping element.

Reply:
Should be proper now.